build.gradle.kts

```diff
@@ -118,6 +133,9 @@ dependencies {
 
     api(project(":vim-engine"))
 
+    ksp(project(":annotation-processors"))
+    implementation(project(":annotation-processors"))
```

Comment:
Could you please note why do we use `implementation` here and not `compile`?

Reply:
Because the `LazyInstance` class is a part of "annotation-processors". I can move it somewhere and make it compile, but I'm not sure where to move it and if it is a good idea

Reply:
Well, I've just made it compileOnly to avoid transitive dependency to kotlinx.serialization